crates/pedersen/Cargo.toml

```
@@ -0,0 +1,37 @@
+[package]
+authors = [
+    "Mirko von Leipzig <mirko.vonleipzig@gmail.com>",
+    "Michael Nielsen <michael@equilibrium.co>",
+    "Krzysztof Lis <klis33@gmail.com>",
+    "Joonas Joonas Koivunen <joonas@equilibrium.co>"
```

```suggestion
    "Joonas Koivunen <joonas@equilibrium.co>"
```

I'd like to keep my second firstname secret :)

---

Ah my bad -- copy paste paste somehow :D